packages/components/src/range-control/index.js

```diff
@@ -15,14 +15,14 @@ import {
 	useState,
 	forwardRef,
 } from '@wordpress/element';
+import * as WPIcons from '@wordpress/icons';
```

Comment:
`import *` should be avoided for the icons package because it breaks tree-shaking causing us to bundle the whole script here. We should only pick the icons we're using.